Std/Util/ProofWanted.lean

```diff
@@ -0,0 +1,28 @@
+import Lean.Elab.Exception
+import Lean.Elab.Command
+import Lean.Parser
+
+
+open Lean Parser Elab Command
+
+/-- This proof would be a welcome contribution to the library!
+
+The syntax of `proof_wanted` declarations is just like that of `theorem`, but without `:=` or the
+proof. Lean checks that `proof_wanted` declarations are well-formed (e.g. it ensures that all the
+mentioned names are in scope, and that the theorem statement is a valid proposition), but they are
+discarded afterwards. This means that they cannot be used as axioms. -/
```

Comment:
Scott's examples resolve this but I would still prefer a sample in the docstring. Since this is new notation, I would like users to have an example to copy-paste rather than think about usage in basic cases.